Sources/XcodeGenKit/InfoPlistGenerator.swift

```diff
@@ -8,20 +8,26 @@ public class InfoPlistGenerator {
      Default info plist attributes taken from:
      /Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/Project Templates/Base/Base_DefinitionsInfoPlist.xctemplate/TemplateInfo.plist
      */
-    var defaultInfoPlist: [String: Any] = {
+    private func generateDefaultInfoPlist(target: Target) -> [String: Any] {
```

Comment:
According to [the naming guideline](https://swift.org/documentation/api-design-guidelines/#parameter-names), I prefer to be more grammatical name like `generateDefaultInfoPlist(for:)`